steps/waldo-upload/0.1.0/step.yml

```diff
@@ -0,0 +1,86 @@
+title: Waldo Upload
+summary: |
+  Uploads a build to Waldo.
+description: |
+  Uploads a build to Waldo.
+website: https://github.com/waldoapp/bitrise-step-waldo-upload
+source_code_url: https://github.com/waldoapp/bitrise-step-waldo-upload
+support_url: https://github.com/waldoapp/bitrise-step-waldo-upload/issues
+published_at: 2020-09-18T16:40:32.952177-04:00
+source:
+  git: https://github.com/waldoapp/bitrise-step-waldo-upload.git
+  commit: 03666f65796c7c9007032faa4a7ec763ca71ad38
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- react-native
```

Comment:
Why not flutter and ionic?

Reply:
Fair enough! We don’t currently have any customers on those platforms (nor on tamarin or cordova), but who knows? Maybe this will inspire new customers. I will add them.